pkg/client/tsconfig.json

```diff
@@ -29,6 +29,7 @@
     },
     "skipLibCheck": true,
     "strict": true, // Includes alwaysStrict and noImplicitAny
+    "noImplicitAny": false,
```

Comment:
Not related to this PR.

Reply:
+1 I will disable ts for the line in question rather than for the whole project.